velox/common/memory/tests/MemoryPoolTest.cpp

```diff
@@ -71,6 +72,7 @@ class MemoryPoolTest : public testing::TestWithParam<TestParam> {
  protected:
   static void SetUpTestCase() {
     FLAGS_velox_memory_leak_check_enabled = true;
+    FLAGS_velox_memory_debug_mode_enabled = true;
```

Comment:
Shall we skip this for concurrent tests otherwise it will be too slow?

Reply:
Actually just found a bug in AsyncDataCache by running this debug mode in concurrent test. It's not too slow actually. It provides values.